AmazonChimeSDK/AmazonChimeSDK/ingestion/IngestionConfigurationBuilder.swift

```diff
@@ -32,7 +32,7 @@ import Foundation
         return self
     }
 
-    public func build(disabled: Bool = true,
```

Comment:
What's this for?

Reply:
enables ingestion by default

Reply:
you mean when the field is not explicitly set to true? just curious how is this related to this bug fix?

Reply:
right. It's not related to this change, but I think we can tack it on. Android SDK already emits ingestion metrics by default.